os/pm/Kconfig

```diff
@@ -123,6 +123,13 @@ config PM_COEF5
 
 		Ignored if CONFIG_PM_MEMORY <= 5
 
+config IDLE_PM
+	bool "TIZENRT IDLE PM framework"
+	default n
+	depends on S5J_HAVE_IDLE_PM
```

Comment:
It is not only for S5J.. It should be for all SoC.
 May be ARCH_HAVE_IDLE_PM would be good. Please check on how other SoC configs are provided